Dockerfile_cc_for_arm

```diff
@@ -35,6 +41,9 @@ RUN python3 -m pip install -U \
     rosdep \
     vcstool
 
+RUN python3 -m pip install -U \
+  git+https://github.com/lark-parser/lark.git@0.7d
```

Comment:
Wouldn't it be better to use the released version from PyPI, i.e. just `pip3 install lark-parser`?

Reply:
That was the version used in the ROS2 building instructions some time ago.
However now it's changed to 

```
apt install -y python3-lark-parser
```

I will update accordingly

https://index.ros.org/doc/ros2/Installation/Linux-Development-Setup/#install-development-tools-and-ros-tools